src/website/app/demos/Navigation/common/navPropDocs.js

```diff
@@ -0,0 +1,81 @@
+/* @flow */
+import { ALIGN } from '../../../../../library/Navigation/constants';
+import formatObject from '../../../utils/formatObject';
+import joinQuoted from '../../../utils/joinQuoted';
+
+import type { ComponentPropDocs } from '../../../pages/ComponentDoc/types';
+
+type NavPropDocs = (
+  component: $FlowFixMe, // Should be React$ComponentType<*>, I think
+  componentName: string
+) => ComponentPropDocs;
+
+const messagesObjectType = {
+  moreLabel: 'string',
+  moreText: 'string'
+};
+
+const navPropDocs: NavPropDocs = (component, componentName) => ({
+  align: {
+    description: 'Horizontal alignment of items',
+    type: {
+      name: 'union',
+      value: joinQuoted(Object.values(ALIGN))
+    },
+    defaultValue: `'${component.defaultProps.align}'`
+  },
+  children: {
+    description: `Content of ${componentName}, most likely NavItems
```

Comment:
Is this always `NavItems` now?

Reply:
Yeah, or styled NavItems, but I think we don't call that out for other component docs.